src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -199,6 +199,16 @@ public function storeHasAProductPricedAt($productName, $price = 100, ChannelInte
     }
 
     /**
+     * @Given /^the store(?:| also) has a product "([^"]+)" originally priced at ("[^"]+")$/
+     */
+    public function storeHasAProductOriginallyPricedAt($productName, $originalPrice = 100, ChannelInterface $channel = null)
```

Comment:
The channel parameter is not needed. 

Also, if you will use the proposal from one comment above, we can use a shared storage transformer.
```php
     /**
       * @Given /^(this product) has been originally priced at ("[^"]+")$/
       */
      public function thisProductIsAlsoPricedAtInChannel(ProductInterface $product, $price)
     {
         /** @var ProductVariantInterface $productVariant */
         $productVariant = $this->defaultVariantResolver->getVariant($product);

         // For all channel pricings for example
 
         $this->objectManager->flush();
     }
```